components/plugin-manager/implementation/src/main/kotlin/PluginEvent.kt

```diff
@@ -23,7 +23,7 @@ import kotlinx.datetime.Instant
 import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
 
-internal data class PluginEvent(
+data class PluginEvent(
```

Comment:
Is is really necessary to remove the `internal` classifiers?

Reply:
Similar below.

Reply:
Yes, because the event and payload classes are part of the `PluginEventStore` API which has to be public.